index.js

```diff
@@ -0,0 +1,41 @@
+import { bookShow, formShow, contactShow } from './modules/navMenu.js';
+
+import Books from './modules/book.js';
+// import {bookAdder} from './modules/addBook.js'
```

Comment:
- [ ] As mentioned in the best practices we should avoid committing commented code to the codebase, I would suggest to remove this commented code from the codebase